core/controllers/library.py

```diff
@@ -116,21 +116,22 @@ def get(self):
             summary_dicts_by_category.insert(0, {
                 'activity_summary_dicts': recently_published_summary_dicts,
                 'categories': [],
-                'header': feconf.LIBRARY_CATEGORY_RECENTLY_PUBLISHED,
+                'header_i18n_id': feconf.LIBRARY_CATEGORY_RECENTLY_PUBLISHED,
                 'has_full_results_page': True,
             })
         if top_rated_activity_summary_dicts:
             summary_dicts_by_category.insert(0, {
                 'activity_summary_dicts': top_rated_activity_summary_dicts,
                 'categories': [],
-                'header': feconf.LIBRARY_CATEGORY_TOP_RATED_EXPLORATIONS,
+                'header_i18n_id':
+                feconf.LIBRARY_CATEGORY_TOP_RATED_EXPLORATIONS,
```

Comment:
This can't go on the same line? If not, do:
```
                'header_i18n_id': (
                     feconf.LIBRARY_CATEGORY_TOP_RATED_EXPLORATIONS),
```

instead.